### PR TITLE
Uxd v9 theme colors

### PR DESCRIPTION
**Pull request by lauralexis**
### Description
Updates v9 light theme, removing recently added hex values, returning to scss expressions to generate values, and updates the theme gray colors.

### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] All tests pass
  - [ ] `yarn lint`
  - [ ] `yarn test-unit`
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by virajsanghvi** [comment]
Closing - will pick this up these changes in another pr

File: src/themes/v9/global_styling/variables/_form.scss [hunk 1]
```diff
@@ -50,12 +50,12 @@ $ouiSwitchColors: (
 ) !default;
 
 // Coloring
-$ouiFormBackgroundColor: $ouiColorEmptyShade !default;
-$ouiFormBackgroundDisabledColor: $ouiColorLightestShade !default;
-$ouiFormBackgroundReadOnlyColor: $ouiColorEmptyShade !default;
+$ouiFormBackgroundColor: tintOrShade($ouiColorLightestShade, 60%, 40%) !default;
+$ouiFormBackgroundDisabledColor: darken($ouiColorLightestShade, 2%) !default;
+$ouiFormBackgroundReadOnlyColor: transparentize(lightOrDarkTheme($ouiColorLightShade, $ouiColorInk), .95) !default;
```
Comment (virajsanghvi) [inline src/themes/v9/global_styling/variables/_form.scss:55]: Have we tried this out on both light/dark theme to validate?

File: src/themes/v9/global_styling/variables/_form.scss [hunk 2]
```diff
@@ -50,12 +50,12 @@ $ouiSwitchColors: (
...
 $ouiFormBorderOpaqueColor: shadeOrTint(desaturate(adjust-hue($ouiColorPrimary, 22), 22.95), 26%, 100%) !default;
-$ouiFormBorderColor: lightOrDarkTheme(transparentize($ouiFormBorderOpaqueColor, .9), $ouiColorLightShade) !default;
-$ouiFormBorderDisabledColor: lightOrDarkTheme(transparentize($ouiFormBorderOpaqueColor, .9), $ouiColorLightestShade) !default;
+$ouiFormBorderColor: transparentize($ouiFormBorderOpaqueColor, .9) !default;
+$ouiFormBorderDisabledColor: transparentize($ouiFormBorderOpaqueColor, .9) !default;
```
Comment (virajsanghvi) [inline src/themes/v9/global_styling/variables/_form.scss:58]: Should this just be $ouiFormBorderColor?

File: src/themes/v9/global_styling/variables/_form.scss (hunk 1 above)
Comment (lauralexis) [reply src/themes/v9/global_styling/variables/_form.scss:55]: Yes, validated in OUI, not OSD, but this is reverting back to the expression that was there prior to the last round of changes.

File: src/themes/v9/global_styling/variables/_form.scss (hunk 2 above)
Comment (lauralexis) [reply src/themes/v9/global_styling/variables/_form.scss:58]: Yes, that works

**Pull request closed without merging**